### PR TITLE
Modals Prototype

### PR DESCRIPTION
**Pull request by martypowell**
- Added "/modals" prototype page
  - Default modal example which can close through a close button or selecting outside of the modal
  - Alternate modal that will only close through a close button click

### CODE REVIEW

File: src/pages/ModalPage.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  Breadcrumbs,
+  BreadcrumbItem,
+  BreadcrumbLinkItem,
+  IconHeading,
+  IconButton,
+  Section,
+  SummaryList
+} from "@baltimorecounty/dotgov-components";
+
+const sideBarContent = [
```
Comment (martypowell) [inline src/pages/ModalPage.js]: would be good to move this into it's own component, since this is repeated so much.

Comment (martypowell) [reply src/pages/ModalPage.js]: this was addressed, other pages will be updated as they are updated

**Review by martypowell — COMMENTED** [review]
left a comment for me

**Comment by sgrg1** [comment]
@martypowell @tmccoy529 , prototyp Modals looks good to me. Functionality worked on Chrome.

**Comment by martypowell** [comment]
@tmccoy529 please take a look at this one, it looks like maybe we did some overlap work here.

**Pull request merged**